src/Tokenizer.js

```diff
@@ -180,14 +149,15 @@ module.exports = class Tokenizer {
     }
   }
 
-  blockquote(src) {
+  blockquote(src, top) {
```

Comment:
should `top` be in the lexer state instead of passing it?

Reply:
Yeah probably.

Reply:
I added this as well. Right now I just set `top` back to `true` at the end of `blockTokens` but this assumes you can only go one level deep, and this passes all the tests. Is there ever a case where you can get more than one level deep? Should `top` instead be `depth` and we add and subtract from it as we go down and up instead of setting true or false?